osu.Game/Database/BackgroundDataStoreProcessor.cs

```diff
@@ -86,6 +92,69 @@ protected override void LoadComplete()
             });
         }
 
+        /// <summary>
+        /// Go through every beatmap and calculate the audio normalization values if they are missing.
+        /// </summary>
+        private void processAudioNormalization()
+        {
+            int beatmapsCount = 0;
+            int beatmapsToProcess = 0;
+
+            realmAccess.Run(r =>
+            {
+                foreach (BeatmapSetInfo beatmapSetInfo in r.All<BeatmapSetInfo>().Where(b => !b.DeletePending))
+                {
+                    beatmapsCount += beatmapSetInfo.Beatmaps.Count;
+                    beatmapsToProcess += beatmapSetInfo.Beatmaps.Count(b => b.AudioNormalization == null);
+                }
+            });
+
+            Logger.Log("beatmaps to process: " + beatmapsToProcess);
+            if (beatmapsToProcess == 0)
+            {
+                Logger.Log("No audio normalization processing required.");
+                return;
+            }
+
+            var notification = showProgressNotification(beatmapsCount, "Verifying loudness level for beatmaps", "all loudness levels have been verified");
+            int processedCount = 0;
+
+            realmAccess.Run(r =>
+            {
+                r.Refresh();
```

Comment:
What's this? No other implementation in this file does this.

Reply:
Remnants of desperate debugging to try and fix a memory leak. Probably shouldn't be there though